ui/src/main/js/view/templates/pipeline-staged.hbs

```diff
@@ -1,7 +1,7 @@
 <div id="pipeline-box">
 <h2>{{fragCaption}}</h2>
 {{#if numStages}}
-<div class="table-box"><div class="table-viewPort" style="max-width:{{maxTableEms}}em">
+<div class="table-box"><div class="table-viewPort">
 <table class="jobsTable" style="max-width:{{maxTableEms}}em">
```

Comment:
This is actually set here: https://github.com/jenkinsci/pipeline-stage-view-plugin/blob/master/ui/src/main/js/view/pipeline-staged.js#L61

But in truth, I think it may not be needed -- seems to play fine with very wide tables (you simply scroll right).